cli/admin/src/test/java/io/pravega/cli/admin/dataRecovery/DataRecoveryTest.java

```diff
@@ -963,6 +963,58 @@ public void testTableSegmentRecoveryCommand() throws Exception {
         Assert.assertArrayEquals(Files.readAllBytes(Paths.get(pravegaStorageDir.toString(), listOfFiles.get(0).getName())), serializedEntries.getCopy());
     }
 
+    @Test
```

Comment:
A verification of this command by manually inducing corruption and then recovering using this command would help.

Reply:
verified table segment-recovery on a pks cluster with these changes and recovery was successful.